graphix/clifford.py

```diff
@@ -37,19 +39,40 @@ class Domains:
 class Clifford:
```

Comment:
`Clifford` can be a frozen dataclass with the single field `index`.

Reply:
I updated the code to use `Enum` here.